src/components/views/voip/VideoView.js

```diff
@@ -50,7 +50,10 @@ module.exports = React.createClass({
     },
 
     getRemoteAudioElement: function() {
-        return this.refs.remoteAudio;
+        // this needs to be somewhere at the top of the DOM which
+        // always exists to avoid audio interruptions.
+        // Might as well just use DOM.
+        return document.getElementById("remoteAudio");
```

Comment:
I don't think this works as-is, and even if it did, I would prefer if we didn't do this.

Reviewing this, I'm trying to find where this element is specified. It isn't in this file. I happen to know it's in the base HTML page which is loaded with Vector, meaning this element isn't even in this project. It's in [vector-web](https://github.com/vector-im/vector-web/blob/master/vector/index.html), but there isn't an `<audio>` tag there with that ID.

This is why I think this doesn't work (it won't find the element). This is also why I don't want to do this (because where the element is defined is so far away from where it is being used, it's incredibly error-prone to get this right).